CHANGES.md

```diff
@@ -82,7 +87,11 @@ To be released.
 [#1102]: https://github.com/planetarium/libplanet/pull/1102
 [#1110]: https://github.com/planetarium/libplanet/pull/1110
 [#1112]: https://github.com/planetarium/libplanet/pull/1112
+<<<<<<< HEAD
 [#1119]: https://github.com/planetarium/libplanet/pull/1119
+=======
+[#1116]: https://github.com/planetarium/libplanet/pull/1116
+>>>>>>> 6137f0014... Changes
```

Comment:
It looks like you have some mistake when rebasing PR.

Reply:
Amended. (https://github.com/planetarium/libplanet/pull/1116/commits/22eae2ac2c598488d38c3684d5b1e2d300149b46)